utils/_context/_scenarios/__init__.py

```diff
@@ -530,6 +530,19 @@ class _Scenarios:
 
     parametric = ParametricScenario("PARAMETRIC", doc="WIP")
 
+    debugger_code_origins = EndToEndScenario(
```

Comment:
There are some scenario with `DD_DYNAMIC_INSTRUMENTATION_ENABLED=1` and `DD_REMOTE_CONFIG_ENABLED=true`, is it possible to use one of them ?